Unique network flows use-case [10552] (#235)
eProsima/Fast-DDS-docs: docs/fastdds/use_cases/use_cases.rst

  Under certain constraints, *Fast DDS* can provide application level communication
  between publishing and subscribing nodes avoiding any data copy during the process.

+ :ref:`use-case-unique-flows`.

>   APIs allowing the identification of the network flows being used, and requesting them to be unique,
>   are explained here.

Reviewer: Although the sentence is completely correct, I'd go for something more direct (not accounting for line length):
```suggestion
  This use case illustrates the APIs that allow for the request of unique network flows, and for the identification of those in use.
```
